commcare-support-library/build.gradle

```diff
@@ -43,8 +43,6 @@ android {
     defaultConfig {
         minSdkVersion 16
         targetSdkVersion 30
-        versionCode 1
```

Comment:
Is this deprecated ?

Reply:
Not really sure why I removed it. I think the library modules don't need the versionCode property, but I can add it back if required?

Reply:
yeah would be good to add these back just in case if it's getting used in the maven publishing.

Reply:
going to leave it as it is since the versionName here is anyway wrong